htdocs/js/wui.js

```diff
@@ -551,11 +551,19 @@ vz.wui.initEvents = function() {
  * @param ev either click event or literal button event value
  */
 vz.wui.handleControls = function(action, keepPeriodStartFixed) {
-	var delta = vz.options.plot.xaxis.max - vz.options.plot.xaxis.min,
-			middle = vz.options.plot.xaxis.min + delta/2,
-			startOfPeriodLocale;
-
+	var delta   = vz.options.plot.xaxis.max - vz.options.plot.xaxis.min;
+	var middle  = vz.options.plot.xaxis.min + delta/2;
+	var hover;
+	var startOfPeriodLocale;
 	var control = typeof action == 'string' ? action : $(this).val();
+	var axes    = vz.plot.getAxes();
+
+	if (vz.options.plot.hoverPos &&
```

Comment:
Where does that option come from?

Reply:
Do you mean the get Axes or the hoverPos?

I have had a look how the **vz.wui.updateHeadline** find out if the hover is within the chart:
https://github.com/volkszaehler/volkszaehler.org/blob/843fb56d96c9f1ae4d8348d03818f703b637e20d/htdocs/js/plot.js#L64-L67

The hoverPos is comming from here:
https://github.com/volkszaehler/volkszaehler.org/blob/843fb56d96c9f1ae4d8348d03818f703b637e20d/htdocs/js/wui.js#L535-L536